src/app/genes/gene-overview/gene-druggability/gene-druggability.component.ts

```diff
@@ -36,6 +36,9 @@ export class GeneDruggabilityComponent implements OnInit {
     classes: string[] = ['A', 'B', 'C', 'D', 'E', 'F'];
     additionalResources: GeneResourceType[] = [];
 
+    noDataTitle = 'Not analyzed';
```

Comment:
These could also be constants.  Looks like they are defined as class variables, but I don't think I've seen this kind of declaration before (without the var/let/const keyword).

Reply:
That seems to be a feature of typescript class members (see e.g. https://stackoverflow.com/questions/37265275/how-to-implement-class-constants).  I switched to using get methods to avoid the entire issue here!